cmd/podman/run.go

```diff
@@ -72,12 +72,23 @@ func runCmd(c *cli.Context) error {
 	options = append(options, libpod.WithUser(createConfig.User))
 	options = append(options, libpod.WithShmDir(createConfig.ShmDir))
 	options = append(options, libpod.WithShmSize(createConfig.Resources.ShmSize))
+
+	// Default used if not overridden on command line
+
+	if createConfig.CgroupParent != "" {
+		options = append(options, libpod.WithCgroupParent(createConfig.CgroupParent))
+	}
+
 	ctr, err := runtime.NewContainer(runtimeSpec, options...)
 	if err != nil {
 		return err
 	}
 
 	logrus.Debug("new container created ", ctr.ID())
```

Comment:
Wait, shouldn't this be a `Debugf()`
@jwhonce You didn't break this, but since you're changing stuff in the neighborhood, would you mind fixing it?

Reply:
Done